src/eval/fixpoint.rs

```diff
@@ -1,8 +1,25 @@
 //! Compute the fixpoint of a recursive record.
 use super::*;
 use crate::position::TermPos;
-use crate::term::{Traverse, TraverseOrder};
-use crate::types::{TypeF, Types};
+
+fn patch_term<C: Cache>(
```

Comment:
Fair remark :slightly_smiling_face:

Reply:
(although it's not part of the public API)

Reply:
Done in [27a96b8](https://github.com/tweag/nickel/pull/1086/commits/27a96b8cec33a24e489837bba27e81e037462391)

Reply:
Oh, I see. I mentioned that because I couldn't  understand what the functions was supposed to do by just looking at the code in the body. Maybe I'm not just familiar with the technical details 😅

Reply:
That's an entirely valid reason for requesting more documentation! I tend to not automatically document small local helpers when they're obvious (from the name) and just a way of breaking implementation down into smaller pieces, but those patching functions are obviously not trivial :sweat_smile: